doc/source/serve/autoscaling-guide.md

```diff
@@ -0,0 +1,369 @@
+        min_replicas: 0
+        initial_replicas: 0
+        max_replicas: 200
+        upscale_delay_s: 30
+        downscale_delay_s: 600
+```
+
+We use [Locust](https://locust.io/) to run a load test against this application. The Locust load test will run a certain number of "users" that ping the Resnet50 service, where each user has a [constant wait time](https://docs.locust.io/en/stable/writing-a-locustfile.html#wait-time-attribute) of 0. This means each user will (repeatedly) send a request, wait for a response, then immediately send the next request. The number of users running over time is shown in the following graph:
+
+![users](https://raw.githubusercontent.com/ray-project/images/master/docs/serve/autoscaling-guide/resnet50_users.png)
+
+The results of the load test are as follows:
+
+::::{tab-set}
+:::{tab-item} Replicas
+<img src="https://raw.githubusercontent.com/ray-project/images/master/docs/serve/autoscaling-guide/resnet50_replicas.png" alt="replicas" width="400"/>
+:::
+
+:::{tab-item} QPS
+<img src="https://raw.githubusercontent.com/ray-project/images/master/docs/serve/autoscaling-guide/resnet50_rps.svg" alt="qps" width="400"/>
+:::
+
+:::{tab-item} Latency
+<img src="https://raw.githubusercontent.com/ray-project/images/master/docs/serve/autoscaling-guide/resnet50_latency.svg" alt="latency" width="400"/>
+:::
+::::
+
+The results of the load test are as follows:
+
+|  |  |  |
+| -------- | --- | ------- |
+| Replicas | <img src="https://raw.githubusercontent.com/ray-project/images/master/docs/serve/autoscaling-guide/resnet50_replicas.png" alt="replicas" width="450"/> |
+| QPS | <img src="https://raw.githubusercontent.com/ray-project/images/master/docs/serve/autoscaling-guide/resnet50_rps.svg" alt="qps" width="450"/> |
+| P50 Latency | <img src="https://raw.githubusercontent.com/ray-project/images/master/docs/serve/autoscaling-guide/resnet50_latency.svg" alt="latency" width="450"/> |
+
+
+<!-- | ![replicas](https://raw.githubusercontent.com/ray-project/images/master/docs/serve/autoscaling-guide/resnet50_replicas.png){width=200px} | ![qps](https://raw.githubusercontent.com/ray-project/images/master/docs/serve/autoscaling-guide/resnet50_rps.png){width=200px} | ![latency](https://raw.githubusercontent.com/ray-project/images/master/docs/serve/autoscaling-guide/resnet50_latency.svg){width=200px} | -->
+
+Notice that the number of autoscaled replicas closely matches the number of Locust users over time. This makes sense because you can simplify it to one user is mapped to one deployment replica, and the user will continuously send the right amount of traffic to keep one replica busy. As expected, the throughput of the system also increases with the number of users and replicas. Lastly, the latency sees some spikes when the traffic increases, but otherwise stays relatively steady. Notice that the biggest latency spike happens at the beginning of the test, because the deployment starts with 0 replicas.
```

Comment:
```suggestion
Note the following:
- Each Locust user constantly sends a single request and waits for a response. As a result, the number of autoscaled replicas closely matches the number of Locust users over time as Serve attempts to satisfy the `target_num_ongoing_requests_per_replica=1` setting.
- The throughput of the system increases with the number of users and replicas.
- The latency briefly spikes when traffic increases, but otherwise stays relatively steady. Note that the biggest latency spike happens at the beginning of the test, because the deployment starts with 0 replicas.
```